apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -66,6 +67,18 @@ export const useValidations: UseValidations = (settings = {}) => {
         },
     }
 
+    const specCharValidator: Rule = {
+        validator: (_, value) => {
+            if (value) {
+                const invalidCharRegex = /[~!@#±$"№%^&*()_+=[\]{}|\\;’:”,./<>?£]/
```

Comment:
Can this regex be reused in the future? May be move it to constant?

Reply:
Maybe, 
```
/[^A-ZА-Я0-9\s\-]/i
```

Reply:
Yes we can, I'll take it out in a constant